### PR TITLE
762245453 modules cms content extra features

### PR DESCRIPTION
**Pull request by orthener**
https://github.com/Overdose-Digital/magento2-cms-management/issues/10

- separate import/export
Overdose/CMSContent/Model/Import.php
Overdose/CMSContent/Model/Export.php
Overdose/CMSContent/Api/ContentImportInterface.php
Overdose/CMSContent/Api/ContentExportInterface.php
- add export to xml
- add export each CMS entity to separate files in one archive
- add import to xml
- fix import to json
- refactoring (adjusted the code for PHP 7.3 version)
- added bulk import from config folder (by cli or s:up)
- fixed few issues with pages import/export (cases with pages that contain blocks)

### CODE REVIEW

**Comment by onlinebizsoft** [comment]
@orthener great to see :)

File: Api/CmsEntityConverterInterface.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
```
Comment (KSergiy) [inline Api/CmsEntityConverterInterface.php]: Need to use strict_type declaration in there and in the next files.

declare(strict_types=1);

File: Api/CmsEntityConverterInterface.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+namespace Overdose\CMSContent\Api;
+
+interface CmsEntityConverterInterface
+{
+    const CMS_MEDIA_NODE = 'media';
+    const PAGE_ENTITY_CODE = 'pages';
+    const BLOCK_ENTITY_CODE = 'blocks';
+
+    /**
+     * @return string
```
Comment (KSergiy) [inline Api/CmsEntityConverterInterface.php]: Need to describe what this function must do ( correct description must be everywhere )

File: Api/CmsEntityConverterInterface.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    public function getCmsEntityType(): string;
+
+    /**
+     * @return string
+     */
+    public function getCmsEntityCode(): string;
+
+    /**
+     * @param array $cmsEntities
+     * @return array
```
Comment (KSergiy) [inline Api/CmsEntityConverterInterface.php]: According to standards between: `@return` and `@param` must be one blank line:
`
* @param something
*
* @return something
`

**Comment by onlinebizsoft** [comment]
Nice progress!

**Pull request merged**